packages/core/src/page/handler/PageHandlerRegistry.ts

```diff
@@ -80,13 +68,16 @@ export default class PageHandlerRegistry extends PageHandler {
 
   /**
    * Executes the post-manage handlers with given arguments
-   *
-   * @param {?ManagedPage} managedPage
-   * @param {?ManagedPage} previousManagedPage
-   * @param {{ type: string, event: Event, url: string }} action
-   * @return {Promise<any>}
    */
-  handlePostManagedState(managedPage, previousManagedPage, action) {
+  handlePostManagedState(
+    managedPage: ManagedPage | null,
+    previousManagedPage: ManagedPage,
+    action: PageAction
+  ) {
+    if (!this._postManageHandlers) {
```

Comment:
I would change it to optional chaining because it can't happen in program  https://github.com/seznam/ima/blob/master/packages/core/src/config/services.js#L32